bitbots_quintic_walk/docs/conf.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
```

Comment:
@ftsell Should this file be added when no changes were made?

Reply:
In my opinion it should be, because then we can track package specific changes like they currently do in bitbtos_vision.